tests/TestCase.php

```diff
@@ -65,4 +66,11 @@ public function call($method, $uri, $parameters = [], $cookies = [], $files = []
 
         return parent::call($method, $uri, $parameters, $cookies, $files, $server, $content);
     }
+
+    public function json($method, $uri, array $data = [], array $headers = [], $options = 0, ?VersionEnum $apiVersion = null): TestResponse
```

Comment:
let's move it to the LaravelHelpers package

Reply:
Done.